app_dart/lib/src/model/appengine/task.dart

```diff
@@ -140,6 +141,11 @@ class Task extends Model {
   @JsonKey(name: 'BuildNumber')
   int buildNumber;
 
+  /// The build number list (reruns) of luci build.
```

Comment:
Can we expand this doc a little bit more to explain the structure and one example?

Reply:
Done.